client/blocks/reader-related-card-v2/docs/example.jsx

```diff
@@ -73,6 +83,9 @@ const moreOnWordPress = [
 			short_excerpt: "This presidential race has, to be blunt, a cluster bomb of attacks, lies and outright nonsense rhetoric that has astounded many and made people curse at their…",
 			canonical_image: {
 					uri: "https://freedomsandtruth.files.wordpress.com/2016/09/trump-rally-in-vegas-getty-640x480.jpg?w=720&quality=80&strip=info",
+			},
+			canonical_media: {
+					src: "https://freedomsandtruth.files.wordpress.com/2016/09/trump-rally-in-vegas-getty-640x480.jpg?w=720&quality=80&strip=info",
```

Comment:
I have some example posts in `blocks/reader-post-card/docs/fixtures.js` you can pull in if you like. I'm currently using those for post card and combined card examples. 

One post still has political content, but that's easily fixed (and will then be gone from all those devdocs examples).